dso-l1/src/main/java/com/tc/object/ClientEntityManagerImpl.java

```diff
@@ -97,6 +99,7 @@
   private boolean isShutdown = false;
   private final boolean reconnectable;
 
+  private final ExecutorService endpointCloser = Executors.newCachedThreadPool();
```

Comment:
i don't think so.  We do not want a delay is starting execution of this task.  I think however, that multiple calls to the same release should return the same future.  Will fix.